### DISCUSSION TITLE
how to install v4.1 kitchen app

### DISCUSSION THREAD

**Discussion by james-ep01** [clientIO/joint#2826] [Q&A]
### Introduction

How do I install the KitchenSink app in version 4.1?
I have to check what is the difference between the code used by KitchenSink in 4.1 and the code currently being developed in version 4.0.x.

To check KitchenSink in version 4.1, I ran the npm install command and put it in apache web server htdoc, but I can't check it. The old 4.0.x was just checking, but.. I think something got complicated.
It would have been nice if there were KitchenSink examples in html and css as before.

What should I do?

And Can I just get the html, css, and js files like the old version?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
We need to update the `README.md`.

> What should I do?

You can run the JS KitchenSink application with:
```sh
npm install
npm start
```

> And Can I just get the html, css, and js files like the old version?

Unfortunately no. But you can convert the ESM modules to a global namespace e.g `app` as it was done in _v3.7_.
